pkg/operator/ceph/object/user/controller.go

```diff
@@ -40,6 +44,22 @@ const (
 
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-object")
 
+const (
+	waitDuration = time.Minute * 1
+	waitFactor   = 2
+	waitJitter   = 0.5
+	waitSteps    = 4
+	waitCap      = time.Minute * 15
+)
+
+var backoff = wait.Backoff{
```

Comment:
I don't see a need to define these as unexported constants and only use them in this one place. I would rather see `var backoff = wait.Backoff{` set directly. Additionally, it would be good to note  what the effect of the options selected is intended to be for those not familiar with all the Backoff parameters.

Reply:
Even my understanding is not so clear (to document better). Any suggestions would be appreciated.

Reply:
- Sounds good to define this `backoff` var inside the method instead of as a package variable
- It would be nice if there were several retries within the first minute as well since it's likely to succeed in that timeframe if the object store CR is created at the same time. Perhaps just changing the duration will be sufficient: `Duration: time.Second * 15`
- You can find the docs here on the backoff params. If you could just add a comment about what these mean it would be helpful. https://godoc.org/k8s.io/apimachinery/pkg/util/wait#Backoff

Reply:
Refined the params and added comments.